src/Nordea/Components/TextInput.elm

```diff
@@ -89,25 +75,36 @@ getAttributes config =
         , config.placeholder |> Maybe.map placeholder
         ]
 
-
-
 -- STYLES
-
-
-styles : List Style
-styles =
+getStyles : Config msg -> List Style
+getStyles config =
+    let
+        borderColorNormal =
+            borderColor Colors.grayMedium
+        borderColorStyle =
+            case config.error of
+                Just error ->
+                    if error then borderColor Colors.redDark
+                    else borderColorNormal
+
+                Nothing ->
+                    borderColorNormal
+
+    in
     [ fontSize (rem 1)
-    , height (em 2.5)
-    , padding2 (em 0.75) (em 0.75)
-    , borderRadius (em 0.125)
-    , border3 (em 0.0625) solid Colors.grayMedium
-    , boxSizing borderBox
-    , width (pct 100)
-    , disabled
-        [ backgroundColor Colors.grayWarm
-        ]
-    , focus
-        [ outline none
-        , borderColor Colors.blueNordea
+        , height (em 2.5)
+        , padding2 (em 0.75) (em 0.75)
+        , borderRadius (em 0.125)
+        , border2 (em 0.0625) solid
+        , borderColorStyle
```

Comment:
```suggestion
        , border3 (em 0.0625) solid borderColor
```

Reply:
I had to keep the variable name, since borderColor is an imported attribute from CSS

Reply:
Yet another reason for not exposing imported variables? (looking at you Css)

Reply:
Haha, I agree. It's also easier to get a quick overview of whats imported and whats our own variables when using Css.blabla instead of just blabla. I think 😊